src/pages/en/core-concepts/framework-components.md

```diff
@@ -5,6 +5,35 @@ description: Learn how to use React, Svelte, etc.
 ---
 Build your Astro website without sacrificing your favorite component framework. Astro supports a variety of popular frameworks including [React](https://reactjs.org/), [Preact](https://preactjs.com/), [Svelte](https://svelte.dev/), [Vue](https://vuejs.org/), [SolidJS](https://www.solidjs.com/), [AlpineJS](https://alpinejs.dev/) and [Lit](https://lit.dev/). 
 
+## Installing Integrations
+
+Astro ships with optional integrations for React, Preact, Svelte, Vue, SolidJS and Lit. One or several of these Astro integrations can be installed and configured in your project.
+
+To configure Astro to use these frameworks, first, install its integration:
+
+```bash
+npm install --save-dev @astrojs/react
+```
```

Comment:
Oh, didn't realize we addressed that! I guess I'm comfortable with this approach, though I do expect many users to ignore peer dependency warnings and our note in the docs (just based on personal experience!). We'll see how it goes though 🤷

Reply:
@bholmesdev But again, to clarify (because I want to make sure everything is accurate), is it correct that they do *not* have to install these things themselves, going off of the other page's React tutorial example and explicit wording? 

I'm not just saying "the warning is there so they'll find it when they don't install the peer dependencies." The Integrations guide explicitly says that, as far as React is concerned for example, they shouldn't even need to worry about it (unless using an old package manager, unless they want a certain version etc. etc.)

Can you confirm that behavior, one way or another, for me?  :)

Reply:
So here's the behavior based on the package manager you use:
- older `npm` versions - peer deps auto-install
- `yarn` - peer deps auto-install
- `pnpm` - peer deps auto-install
- _newer_ `npm` versions - **peer deps don't auto-install**

☝️ This group of plain `npm` users may be surprised to find errors in their console. Here's the error I see when installing `@astrojs/react` with the latest `npm`. Note that it's not very human-readable:
![image](https://user-images.githubusercontent.com/51384119/159299666-6599260f-05b0-4eb9-ae67-80f9dbe96864.png)